### DISCUSSION TITLE
Does Evaluation of CYP1A2 Inhibitor & Inducer mediated DDI Simultaneously

### DISCUSSION THREAD

**Discussion by bhim1992** [Open-Systems-Pharmacology/Forum#1790] [Questions & Problems]
If an investigational drug is both an inducer and inhibitor of CYP1A2, for the combined effect, should we take the same substrate, such as Caffeine, for CYP1A2? As I see in the DDI network evaluation for CYP1A2, Caffeine is the substrate, and the model is validated with fluvoxamine (inhibitor). Separately for induction, the model is evaluated by taking Theophylline as substrate and Rifampin as inducer. In such a case, we can evaluate inhibition with caffeine and induction with Theophylline. However, how should we proceed if we have to evaluate both inhibition and induction? Can we take caffeine as the substrate for both inhibition and induction DDI? 

Thank you in advance for resolving my query.

**Comment by tobiasK2001** [comment]
Hi,

I think you can take theophylline and caffeine as victim substrates for both effects: inhibition and induction. Please make sure to include both processes in your Simulation.

Best, Tobias

**Reply by bhim1992** [reply]
Thank you Tobias, 
Is that means if I demonstrate CYP1A2 inhibit taking caffeine as victim and Fluvoxamine as inhibitor, model can be considered valid to reflect induction process as well? Can I take this caffeine model for evaluation of induction, inhibition and combined process  for test drug? 
Than you once again. 

Best Regard
Bhim
